VideoWeb/VideoWeb/ClientApp/src/app/vh-officer/vho-hearing-list/vho-hearing-list.component.html

```diff
@@ -36,7 +36,6 @@
     </ng-container>
     <div class="title listItemBig" id="{{ conference.id }}-case-name">{{ conference.caseName }}</div>
     <div class="listed listItemGrey" id="{{ conference.id }}-duration">{{ getDuration(conference) }}</div>
-    <div class="crest"><img src="/assets/images/black_crest.png" class="icons" [attr.alt]="'vho-hearing-list.crest-icon-alt' | translate" /></div>
```

Comment:
can we also remove the string content from en.json and cy.json, as they dont appear to be used anywhere else

Reply:
Well spotted :) . Done.